README.md

```diff
@@ -4,6 +4,7 @@
 
 - **Version 2**
 
+  - 2.0.8: added active status to primary and secondary topnav
```

Comment:
Because you've updated two separate components, I'd opt to separate these change log messages, like so:

- 2.0.7:
  - Added active status to PrimaryTopNav
  - Added active status to SecondaryTopNav

Note: also use the full component name.